core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -3994,7 +3995,7 @@ public Coin getBalance(BalanceType balanceType) {
         try {
             if (balanceType == BalanceType.AVAILABLE || balanceType == BalanceType.AVAILABLE_SPENDABLE) {
                 List<TransactionOutput> candidates = calculateAllSpendCandidates(true, balanceType == BalanceType.AVAILABLE_SPENDABLE);
-                CoinSelection selection = coinSelector.select(NetworkParameters.MAX_MONEY, candidates);
+                CoinSelection selection = coinSelector.select(MAX_MONEY, candidates);
```

Comment:
All CoinJoin related changes will be pulled out into a subclass of Wallet in a future PR.

Reply:
Of course this change isn't related to CoinJoin